packages/babel-helper-define-polyfill-provider/src/imports-cache.js

```diff
@@ -1,7 +1,8 @@
 // @flow
 
-import { types as t } from "@babel/core";
 import type { NodePath } from "@babel/traverse";
+import * as babel from "@babel/core";
+const { types: t } = babel.default || babel;
```

Comment:
This is because Rollup's interop is incompatible with Node.js's behavior (https://github.com/rollup/plugins/issues/635)

Reply:
https://github.com/rollup/plugins/pull/838 will make it possible to avoid doing this, if it will get merged.